doc/sphinx/reference/onedim/nonlinear-solver.md

```diff
@@ -1,5 +1,387 @@
+$$
+
+where:
+- $s_{n,j}$ is the Newton step vector component for the $n$-th solution variable
+  at the $j$-th grid point.
+- $w_n$ is the error weight for the $n$-th solution component, given by:
+
+$$
+w_n = \epsilon_{r,n} \cdot \frac{\sum_j |x_{n,j}|}{J} + \epsilon_{a,n}
+$$
+
+Here:
+- $\epsilon_{r,n}$ is the relative error tolerance for the $n$-th solution component.
+- $\frac{\sum_j |x_{n,j}|}{J}$ is the average magnitude of the $n$-th solution
+  component over all grid points, and $J$ is the total number of grid points.
+- \( \epsilon_{a,n} \) is the absolute error tolerance for the $n$-th solution
+  component.
+
+#### Interpretation of the Weighted Norm
+
+The weighted norm is a relative measure that helps bring all components of the step
+vector into a comparable range, taking into account the scales of the different
+solution components. It can be interpreted as follows:
+
+- **Relative Error Term** $(\epsilon_{r,n})$: Scales the step size relative to the
+  average magnitude of the corresponding solution component. This means that larger
+  components can tolerate larger steps.
+- **Absolute Error Term** $(\epsilon_{a,n})$: Ensures that even very small solution
+  components are considered in the convergence check by providing a minimum threshold.
+
+#### Convergence Criterion
+
+The Newton iteration is considered converged when the weighted norm is less than 1:
+
+$$
+\sum_{n,j} \left(\frac{s_{n,j}}{w_n}\right)^2 < 1
+$$
+
+This criterion indicates that each component of the step vector $s_{n,j}$ is
+sufficiently small relative to the expected precision (as defined by the weights
+$w_n$).
+
+## Transient Solution
+
+There will be times when the solution of the steady-state problem can not be found
+using the damped Newton method. In this case, a transient solution is solved and a
+specified number of time steps are taken before the steady-state damped Newton
+method is attempted again.
+
+
+The equation that is being solved for the transient case is:
+
+$$
+\frac{d x}{dt} = F_{ss}(x)
+$$
+
+Where $F_{ss}(x)$ is the residual vector for the steady-state problem. That is, the
+residual vector that arises when the $\frac{d x}{dt}$ term is zero in the governing
+equations. The transient solution is solved using the implicit backward Euler method.
+The solution at the next time step is given by:
+
+$$
+\frac{x_{n+1} - x_n}{\Delta t} = F_{ss}(x_{n+1})
+$$
+
+Here the `n+1` is the solution at the next time step, `n` is the solution at the current
+time step.
+
+We consider a case where each element of the residual vector may not have a
+corresponding time derivative term. These equations without time derivative terms are
+referred to as algebraic equations, and the ones with time derivative terms are
+referred to as differential equations. A general way to express this is by writing the
+equation above in the following form.
+
+$$
+\alpha \frac{x_{n+1} - x_n}{\Delta t} + F_{ss}(x_{n+1})
+$$
```

Comment:
Missing "= 0". Also, isn't there a sign error here, such that it should be
$$F_{ss}(x_{n+1}) - \alpha \frac{x_{n+1} - x_n}{\Delta t} = 0$$?

Reply:
The plus was supposed to be an equals there, as I was just re-writing the equation above it in a more general form.